### ISSUE TITLE
Plugin crash when playing in editor on iOS & Android.

### ISSUE THREAD

**Issue by alamboley** [adjust/unity_sdk#116]
Hey,

On my iOS & Android game, I've added the plugin. When clicking Unity's play button, I've errors.
On iOS:
```EntryPointNotFoundException: _AdjustLaunchApp```
On Android:
```
Exception: JNI: Init'd AndroidJavaClass with null ptr!
Rethrow as TypeInitializationException: An exception was thrown by the type initializer for com.adjust.sdk.AdjustAndroid
```

Is it the expected behavior? Why it isn't inside a #!UNITY_EDITOR flag?

**Comment by uerceg** [comment]
Hi @alamboley 

We have PR submitted to address this issue (https://github.com/adjust/unity_sdk/pull/114) and we will have it live in our next update. Adjust SDK is not meant to run in Unity Editor, so with next update we will simply prevent all SDK logic to be executed in case that `UNITY_EDITOR` flag is active.

SDK logic should execute only in cases of Android, iOS or Windows targets.

**Comment by alamboley** [comment]
Thanks for the answer @uerceg. It's a bit crazy that it's still not encapsulated! Waiting the update.

**Comment by uerceg** [comment]
@alamboley 

This should now be fixed with latest v4.12.2 version: https://github.com/adjust/unity_sdk/releases/tag/v4.12.2

Feel free to reopen the issue if problem still persists with latest SDK version.

Cheers
